src/e2e-test/resources/credentials

```diff
@@ -0,0 +1,13 @@
+{
+  "type": "service_account",
+  "project_id": "cdf-athena",
+  "private_key_id": "b8af3e57ef8220bcc3c4924f68f8af0b69ab17e4",
+  "private_key": "-----BEGIN PRIVATE KEY-----\nMIIEvQIBADANBgkqhkiG9w0BAQEFAASCBKcwggSjAgEAAoIBAQDFNdAexQlIbgJO\nVFbVpxS8ZXefu5AhGdG5DkP1SXvuyXgJEcfWnDQGNG2TL/7Wd5fO6xyVDAvfvBtC\n5vmu/4AC7MNvrwIueq/sisK7Yz+psiYjia6HSJBESX66VANGvCbdPMqiWYGOE81S\nLtCOu2ecjQuPYMUKxkoBlP+U6tQI1sUhladO5LRAlpJ43j26EIurOwwwTMBpFGgy\n1JNml6D9mSldmrvfJ+C0//3lRt05/EveQIJ/afKcJ2S7ZS0i5jSVV0joAtunskFG\noZkbLTIguoLaYPByyHGU3RY4ZpRhGCF9+2MTwGGR6HRt9i8Hx2bBsgf16YxlAwRa\nFdmgtkbBAgMBAAECggEAAO/dZCYIwiG2rFdZfcSgVDhRsg42tDeOj+fl0gKmPndR\n1Q4y8j22ABEL87ryxaNWEu86SKmA3XBj+0SEa2zOl4T3NT7CPoUJ9LBgljoVnXlp\nlbnj3MPvHonwWpOHW/TNJPgy2MRe2ZVVwIYVKtZwWrWvWEAsE33ncFw8PiGC/iM/\n8Ee/squ5T57+2VZvj+z8W5RQid9K61SVlg+pRjiyLutOEjh1Q1C5fyK7qrvcwhm0\nB9DrMGrWs8+J9YeTHFFdSVUBMTNTsR//GgvXqi6mYVkC+AlIAMslttVXuvLNX/mj\nLHn0njmVcFTzzlnFc+N9561R4XljYdRhTZgxtre56QKBgQD9NSIMBcRiAf2VO2OI\nGsda5qzLfTZCbyjt29P5RIgiRXMf7M3U0hI7BDCRw3RD6baUafAzO0RUf2tYeHYc\n7j6BIckCYE4V723D3HH+zg6tmBEmVXgoDNir7dqNLKGhaNs9izd6GWqAZwp6meiI\n20DTull4i1KAbajIsWUFvreEqQKBgQDHYpXzHgbzuIoCab6etzEdZLwyXRr61+Zo\ncOcLe2RbyR2D0ZXvJpTtpEn71O05DHnQ7OeVLorZsuN+yy3X35UDjaAjNs96bc9Z\n7ItOOy8ddR8UvxO1UwGLgB23kSNVmziN8GXmkYy6FNxsedxGkbJr1lho8beJk9xv\nrnK06zDoWQKBgDUJDwOTpamdnx7F3LgiW8VHccQBiv7pR5pipAie+SCsiUuTjx1X\njuf7nay6Urzy1AJMGjqw3EQSmEpkmODAaikkCXVM9K4Vg1k18E6dnGdx5Y2w80T4\nQ93nKRLO2V/0lY9TfKA/eJZCfuhlGlaoDfELJDhifjpWPbcjqPRGGSxZAoGBAMBc\nQMb5ZUOl+qAxXRLLMrtlAJymbu0rdsO0iXPua/iRo8QVZE4CLxTGuAoad3PesXf7\n18F50cQObqVZaLUopB91d9YBf1JitvXO3Odw+he1c3MUlfMOTiNES2daw3hH9Nwb\n3nQAYID2t2KAfyAmWLjpZwp4idZVrQqih1A8yEahAoGATOLUb+AM9aGoTEhQawxH\nIHqMFWaVQoYD1+GispO9ff2RdB80vvY3A+gVydNvZq1bTwKaUYSNlt8nd/L9/72Z\nOvZ2J3acCQqcQ9WziHqaHEXIqMrqY9XjIOn7lmLZCh56vBG9loKkXoV3Psaxxrgq\n+/vnHrPP7d7P27HdcG1fCaE=\n-----END PRIVATE KEY-----\n",
+  "client_email": "github-action@cdf-athena.iam.gserviceaccount.com",
+  "client_id": "100557732933893913758",
+  "auth_uri": "https://accounts.google.com/o/oauth2/auth",
+  "token_uri": "https://oauth2.googleapis.com/token",
+  "auth_provider_x509_cert_url": "https://www.googleapis.com/oauth2/v1/certs",
+  "client_x509_cert_url": "https://www.googleapis.com/robot/v1/metadata/x509/github-action%40cdf-athena.iam.gserviceaccount.com",
+  "universe_domain": "googleapis.com"
+}
```

Comment:
Where we are using this, please don't check in service account JSON and other sensitive info to github repos. Remove the file.